resources/definitions/ultimaker_original.def.json

```diff
@@ -14,12 +14,7 @@
         "has_materials": true,
         "preferred_material": "*pla*",
         "preferred_quality": "*normal*",
-        "pages": [
-            "SelectUpgradedParts",
-            "UpgradeFirmware",
-            "UltimakerCheckup",
-            "BedLeveling"
-        ]
+        "supported_actions":[ "UMOCheckup", "UpgradeFirmware"]
```

Comment:
Oh. Here i was thinking we only wanted bed level for the heated bed. Oh well, it's easy enough to add.